DEVELOPER.md

```diff
@@ -14,10 +14,13 @@ See http://kafka.apache.org/documentation.html#consumerconfigs for details about
 
     input {
         kafka {
-            topic_id => ... # string (required), The topic to consume messages from
+            topic_id => ... # string (optional), default: nil, The topic to consume messages from. Can be a java regular expression for whitelist of topics.
```

Comment:
why do we have `topic_id` to be optional? what does it mean to have `nil` as default? Can we make this required?